### PR TITLE
Unfck A/B test header for German (Fixes #9724)

### LINKED ISSUE #9724: A/B test page header on EU unfck landing pages
## Description

https://www.figma.com/file/w548NlWIvHvEU0igul9U2H/UNFCK-LP-Redesign?node-id=61%3A168

---

#### Success Criteria
- [ ] Experiment is configured via Traffic Cop.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
A/B test header on German unfck landing page:
- http://localhost:8000/de/firefox/unfck/?v=1 (control)
- http://localhost:8000/de/firefox/unfck/?v=2 (alt header)

## Issue / Bugzilla link
#9724

## Testing
Note: test using a non-Firefox browser with DNT disabled.

- [x] When landing on http://localhost:8000/de/firefox/unfck/, Traffic Cop redirects to either of the variations as expected.
- [x] GA `dataLayer.push()` events for each variation look ok.
- [x] Responsiveness of new header variation looks ok.

### CODE REVIEW

**Comment by amychurchwell** [comment]
Sorry I was unable to get to this today, I got tied up with the l10n migration work (which is done now! all that is left are the bedrock PR's) I will pick this up tomorrow AM 👍

File: media/js/firefox/campaign/unfck-de-experiment.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                }
+            });
+
+            cop.init();
+        }
+    };
+
+    // Avoid entering automated tests into random experiments.
+    // Target audience is non-Firefox users.
+    if (href.indexOf('automation=true') === -1 && !isFirefox) {
```
Comment (amychurchwell) [inline media/js/firefox/campaign/unfck-de-experiment.js:39]: IIRC; this is the first time I have seen us opt automated tests out of traffic cop experiments, which is something that hadn't occurred to me. Should we move this logic into the TC library itself, or are there cases where we would want automated tests to be included? (Not for this PR, of course)

**Review by amychurchwell — APPROVED** [review]
lgtm! r+ 🧇

File: media/js/firefox/campaign/unfck-de-experiment.js (hunk 1 above)
Comment (alexgibson) [reply media/js/firefox/campaign/unfck-de-experiment.js:39]: We could do something like this yeah, good suggestion :) For now it's bedrock specific 👍

**Pull request merged**